packages/apps/transfer/src/services/modules/describe-module.ts

```diff
@@ -26,30 +22,24 @@ export const describeModule = async (
   gasPrice: number = kadenaConstants.GAS_PRICE,
   gasLimit: number = kadenaConstants.GAS_LIMIT,
   ttl: number = kadenaConstants.API_TTL,
-): Promise<IModuleResult> => {
+): Promise<ILocalCommandResult> => {
   debug(describeModule.name);
```

Comment:
Maybe include a useful prefix? 
```suggestion
  debug(`retrieving (describe-module "${describeModule.name}")`);
```